browser/src/ReadData/ReadData.tsx

```diff
@@ -288,7 +289,7 @@ class ReadData extends Component<ReadDataProps, ReadDataState> {
 
     const locus = `${chrom}:${start}-${stop}`
 
-    const browserConfig =
+    let browserConfig =
```

Comment:
Instead of making this mutable and then overwriting the whole value after the fact if we have a v4 dataset, let's pull this into a helper function with this general structure:

```
const locus = `${chrom}:${start}-${stop}`

if(isV4(datasetId)) {
  return({ /* config for v4 here */}
}

if(referenceGenome === 'GRCh37) {
  return({ /* config for v2 goes here */ })
}

return ({ /* config for v3 goes here /* })
```

Reply:
Good suggestion -- fixed.